addon/components/horizontal-bar-chart.js

```diff
@@ -116,7 +116,21 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   }),
 
   marginTop: Ember.computed.alias('labelPadding'),
-  marginBottom: Ember.computed.alias('labelPadding'),
+
+  /**
+   * The margin at the bottom depends on the label and title padding and height.
+   * @override
+   * @type {Number}
+   */
+  marginBottom: Ember.computed('labelPadding', 'xTitleVerticalOffset',
+      'hasXAxisTitle', function() {
+    if (this.get('hasXAxisTitle')) {
+      // 15 is the expected height of the text.
+      return this.get('labelPadding') + this.get('xTitleVerticalOffset') + 15;
```

Comment:
This is because we dont have an effective way to measure the exact height of the text. So this is an estimate what what the size of the text is. If user has text with different size, they can override xTitleVerticalOffset to adjust the amount of space we have.